connector/src/main/scala/com/basho/riak/spark/rdd/connector/SessionCache.scala

```diff
@@ -109,7 +117,18 @@ class SessionCache(val cacheSize: Int) extends Logging {
       }
 
       val ns = nodes.toBuffer
-      val cluster = RiakCluster.builder(ns.asJava).build()
+      val cluster = RiakCluster.builder(ns.asJava)
```

Comment:
We need to add comment here to provide an explanation why we need to setup all this stuff